File: docs-old-1104/data-tools/codesnippet/VisualBasic/read-xml-data-into-a-dataset_1.vb
    Private Sub ReadXmlButton_Click() Handles ReadXmlButton.Click

        Dim filePath As String = "Complete path where you saved the XML file"

        AuthorsDataSet.ReadXml(filePath)

        DataGridView1.DataSource = AuthorsDataSet
        DataGridView1.DataMember = "authors"
    End Sub

File: docs/data-tools/codesnippet/CSharp/read-xml-data-into-a-dataset_1.cs
        private void ReadXmlButton_Click(object sender, EventArgs e)
        {
            string filePath = "Complete path where you saved the XML file";

            AuthorsDataSet.ReadXml(filePath);

            dataGridView1.DataSource = AuthorsDataSet;
            dataGridView1.DataMember = "authors";
        }